tests/unit/dash/development/test_base_component.py

```diff
@@ -244,17 +244,6 @@ def test_get_item_raises_key_if_id_doesnt_exist(self):
         with self.assertRaises(KeyError):
             c3['0']
 
-    def test_equality(self):
-        # TODO - Why is this the case? How is == being performed?
-        # __eq__ only needs __getitem__, __iter__, and __len__
-        self.assertTrue(Component() == Component())
-        self.assertTrue(Component() is not Component())
-
-        c1 = Component(id='1')
-        c2 = Component(id='2', children=[Component()])
-        self.assertTrue(c1 == c2)
```

Comment:
Kind of a nice side effect, IMO, that components are now no longer always equal to each other - though I don't know why this would come up...